src/components/calcite-pick-list-group/calcite-pick-list-group.tsx

```diff
@@ -56,11 +56,14 @@ export class CalcitePickListGroup {
     };
 
     const title = groupTitle;
+    const parentLevel = el.closest("calcite-pick-list")?.headingLevel;
+    const relativeLevel = parentLevel ? ConstrainHeadingLevel(parentLevel + 1) : null;
```

Comment:
Unrelated: Since this is a utility function and not a functional component, should this be `constrainHeadingLevel`?